core/common/src/test/java/alluxio/underfs/FingerprintTest.java

```diff
@@ -55,4 +55,33 @@ public void parseInvalidFingerprint() {
     Assert.assertNotNull(expected);
     Assert.assertEquals(expected, Fingerprint.parse(expected).serialize());
   }
+
+  @Test
+  public void matchMetadataTest() {
```

Comment:
perhaps rename to `matchMetadataOrContent` since it tests `matchContent` as much as `matchMetadata`
since we need the `@Test` annotation anyway, we don't suffix our tests with "Test"